packages/client-core/src/admin/components/Instance/PatchInstanceserver.tsx

```diff
@@ -47,25 +50,30 @@ const PatchInstanceserver = ({ open, onClose }: Props) => {
   useEffect(() => {
     if (location.created.value) {
       onClose()
-      setState({
-        ...state,
-        location: ''
-      })
+      state.location.set('')
     }
   }, [location.created])
 
-  const handleChange = (e) => {
-    const { name, value } = e.target
-    setState({ ...state, [name]: value, locationError: value.length < 2 ? 'Location is required!' : '' })
+  const handleChangeLocation = (e) => {
+    const { value } = e.target
+    state.merge({
+      location: value,
+      locationError: value.length < 2 ? 'Location is required!' : ''
+    })
+  }
+
+  const handleChangeCount = (e) => {
+    const { value } = e.target
+    state.count.set(value)
   }
 
   const handleSubmit = () => {
     let locationError = ''
     if (!state.location) {
```

Comment:
Shouldnt it be `state.location.value` instead of `state.location`?

Reply:
yes, good catch